### ISSUE TITLE
Loading PCA model unclear

### ISSUE THREAD

**Issue by adinhobl** [JuliaAI/MLJMultivariateStatsInterface.jl#10]
I've tried to load the PCA model from the MultivariateStats package, which I had in my environment. When I run `info("PCA")` I get the following:
```
Principal component analysis. Learns a linear transformation to
project the data  on a lower dimensional space while preserving most of the initial
variance.

→ based on [MultivariateStats](https://github.com/JuliaStats/MultivariateStats.jl).
→ do `@load PCA pkg="MultivariateStats"` to use the model.
→ do `?PCA` for documentation.
(name = "PCA",
 package_name = "MultivariateStats",
 is_supervised = false,
 docstring = "Principal component analysis. Learns a linear transformation to\nproject the data  on a lower dimensional space while preserving most of the initial\nvariance.\n\n→ based on [MultivariateStats](https://github.com/JuliaStats/MultivariateStats.jl).\n→ do `@load PCA pkg=\"MultivariateStats\"` to use the model.\n→ do `?PCA` for documentation.",
...
```
Next, I run this `@load PCA pkg="MultivariateStats"`, as stated above. This returns
```
PCA(
    maxoutdim = 0,
    method = :auto,
    pratio = 0.99,
    mean = nothing) @329
```
So far, I think I'm clear on all the steps that have happened. Now, I expect to have a model called PCA available.
That's also what I expect from [this example](https://alan-turing-institute.github.io/MLJ.jl/dev/common_mlj_workflows/#Basic-fit/transform-for-unsupervised-models-1 ).

Now, I try to run the next few lines from the example:
```
model = PCA(maxoutdims=2)
mach = machine(model, X_stand)
MLJ.fit!(mach, rows=train)
```
and I recieve the following output:
```
MethodError: no method matching PCA(; maxoutdims=2)
Closest candidates are:
  PCA(!Matched::Array{T,1}, !Matched::Array{T,2}, !Matched::Array{T,1}, !Matched::T) where T<:Real at /home/andrew/.julia/packages/MultivariateStats/BYMwD/src/pca.jl:15 got unsupported keyword argument "maxoutdims"
  PCA(!Matched::Array{T,1}, !Matched::Array{T,2}, !Matched::Array{T,1}, !Matched::T, !Matched::T) where T<:Real at /home/andrew/.julia/packages/MultivariateStats/BYMwD/src/pca.jl:6 got unsupported keyword argument "maxoutdims"

Stacktrace:
 [1] top-level scope at In[37]:1
 [2] include_string(::Function, ::Module, ::String, ::String) at ./loading.jl:1091
 [3] execute_code(::String, ::String) at /home/andrew/.julia/packages/IJulia/rWZ9e/src/execute_request.jl:27
 [4] execute_request(::ZMQ.Socket, ::IJulia.Msg) at /home/andrew/.julia/packages/IJulia/rWZ9e/src/execute_request.jl:86
 [5] #invokelatest#1 at ./essentials.jl:710 [inlined]
 [6] invokelatest at ./essentials.jl:709 [inlined]
 [7] eventloop(::ZMQ.Socket) at /home/andrew/.julia/packages/IJulia/rWZ9e/src/eventloop.jl:8
 [8] (::IJulia.var"#15#18")() at ./task.jl:356
```
I figure this is some kind of name resolution error, so I try the same thing, but with `model = MultivariateStats.PCA()` and receive the same error. Confused, I finally try this 
```
model = MLJMultivariateStatsInterface.PCA()
mach = machine(model, X_stand)
MLJ.fit!(mach, rows=train)
```
and recieve 
```
┌ Info: Training Machine{PCA} @502.
└ @ MLJBase /home/andrew/.julia/packages/MLJBase/Ov46j/src/machines.jl:319

Machine{PCA} @502 trained 1 time.
  args: 
    1:	Source @970 ⏎ `Table{AbstractArray{Continuous,1}}`
```
I was prompted to install this package earlier and figured I would try it, and it looks like it worked.
So I have a few questions:
1. Did I install the model from MultivariateStats I thought I was?
1. Am I correct about it being a name resolution error or did I mis-diagnose the problem? If I did the right thing, it might be clearer to refer to the MLJMultivariateStatsInterface in the info somewhere. 

Thanks! I'm doing projects for Masters courses, and I've generally tried to do it using Julia and MLJ so far and haven't run into many issues. Good work!

**Comment by tlienart** [comment]
I think you did everything fine apart from the typing of X, you gave the machine an array and it expects a *table* (check the MLJ docs for this, the reasoning is that MLJ expects data with a schema, but this is better explained in the docs)

Just use `Xt = MLJ.table(X_stand)` and see if it works?

**Note**: in the example you pointed to, `X` is a DataFrame which is a table.

@ablaom if that's correct do you think the error could be caught and a clearer error message or pointer to docs thrown?

**Comment by adinhobl** [comment]
@tlienart thanks for the response. `X_stand` comes directly from the Standardizer.
```
standardizer = Standardizer(count=true)
stand = machine(standardizer, X[train,:]) #only want to standardize on training data
MLJ.fit!(stand)
X_stand = MLJ.transform(stand, X);
```
It _should_ be a DataFrame.
![image](https://user-images.githubusercontent.com/29109741/97092434-924d7a80-1609-11eb-96c4-000c6c85808b.png)

**Comment by adinhobl** [comment]
Also, if I try `@load PCA pkg="MLJMultivariateStatsInterface"`, it doesn't work. It just feels a little awkward to load the package from `MultivariateStats` and then reference it as `model = MLJMultivariateStatsInterface.PCA()`. I think a message would help if this is the expected way to load and reference PCA.

**Comment by tlienart** [comment]
Sorry that my previous comment was not the right fix, ironically I was misguided by the error message... it seems it is indeed a clash between an imported PCA name and the one that MLJ tries to load. I guess you could you try in a separate Julia session to not do `using MultivariateStats` and see if that helps. But before that, another approach I think is that you can directly get the right model object from the load call:

```
pcamodel = @load PCA pkg="MultivariateStats"
m = machine(pcamodel, X_stand)
```

and then use that in the machine; that should fix your issue in your current session I think.

**Comment by adinhobl** [comment]
Your first suggestion worked - I commented out `using MultivariateStats` and reran the notebook.
![image](https://user-images.githubusercontent.com/29109741/97094960-7228b600-161f-11eb-8683-6d738397a93c.png)

Your second suggestion also worked - I uncommented `using MultivariateStats` so it was active and directly assigned the model to the macro output. However, I do get a warning about a name conflict.
![image](https://user-images.githubusercontent.com/29109741/97095020-0eeb5380-1620-11eb-9363-4c1762e1086f.png)
In this, `PCA` is assigned to the one from the `MultivariateStats` package, and `PCA2` is assigned to `MLJMultivariateStatsInterface`.

I guess it was mostly user error (or cluelessnes). Thanks for your help!
